el8/Dockerfile.buildtime

```diff
@@ -11,6 +11,8 @@ RUN dnf install -y perl-Carp perl-PathTools perl-Data-Dumper perl-Errno perl-Exp
     dnf install -y tar rpm-build git nspr nss rsync wget openssl-devel ncurses-devel libX11-devel \
           readline-devel tcl-devel tk-devel mesa-libGLU-devel libXext-devel libXft-devel \
           libXpm-devel java-1.8.0-openjdk-devel java-11-openjdk-devel libtool gcc-c++ &&\
+    epel_pkgs="@EPEL_PACKAGES@" &&\
+    if [ -n "$epel_pkgs" ] ; then dnf install --enablerepo=epel -y $epel_pkgs; fi &&\
     xcmd="@EXTRA_COMMAND@" &&\
```

Comment:
I think `--enablerepo=epel` is not needed here. `epel` repo is enabled by default.